source/NVDAObjects/UIA/__init__.py

```diff
@@ -2268,15 +2294,10 @@ def event_stateChange(self):
 
 class SearchField(EditableTextWithSuggestions, UIA):
 	"""An edit field that presents suggestions based on a search term.
+	This is now an empty class as functionality has been moved to the base EditableText behaviour.
+	@warning Deprecated, may be removed in future versions.

```

Comment:
Couldn't we just remove this as we're at an API breaking release?

Reply:
The policy now days is to only remove something if its continued existence actually conflicts with or stops a newer improved implementation. Keeping this SearchField class is not harmful.